packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -375,6 +381,22 @@ contract OPContractsManager is ISemver {
         return address(uint160(bytes20(bytes.concat(versionByte, first19Bytes))));
     }
 
+    function computeProxyContractSalt(
+        uint256 _l2ChainId,
+        string memory _saltMixer,
+        string memory _contractName
+    )
+        internal
+        pure
+        returns (bytes32)
+    {
+        return keccak256(abi.encode(_l2ChainId, _saltMixer, _contractName));
+    }
+
+    function computeNonProxiedSalt(uint256 _l2ChainId, string memory _saltMixer) internal pure returns (bytes32) {
+        return keccak256(abi.encode(_l2ChainId, _saltMixer));
+    }
+
```

Comment:
nit: natspec needed here. 

Since the point is to clarify the usage, it would also be good to add a note clarifying why these two different salt types are needed.

Reply:
sg, added natspec in this commit: https://github.com/ethereum-optimism/optimism/pull/13273/commits/333bf545b63ce454215eaa44e5381bfd7d43098b

Reply:
Added a little more color in this commit: https://github.com/ethereum-optimism/optimism/pull/13273/commits/4b081cac0644ea69dd43863af701ad52fbdeb0e0

Reply:
LGTM, resolving